glide.yaml

```diff
@@ -10,7 +10,7 @@ import:
   version: 6d8c23d2e66ce61e2a7e34f92e8eb3eec13279fa
 - package: github.com/coreos/etcd
   version: v3.1.9
-- package: github.com/Sirupsen/logrus
+- package: github.com/sirupsen/logrus
```

Comment:
Do we actually use this package?
The original etcd operator used the Sirupsen package. I liked the idea of structured logging because I think we want to tag log entries so we know which worker they came from. But it seemed like that package didn't support logging filename and linenumber. So I settled on using the Google log package for now.